torch/onnx/utils.py

```diff
@@ -611,6 +618,8 @@ def _run_symbolic_function(g, n, inputs, env, operator_export_type=OperatorExpor
         from torch.onnx.symbolic_helper import _export_onnx_opset_version as opset_version
         import torch.onnx.symbolic_registry as sym_registry
 
+        if operator_export_type == OperatorExportTypes.ONNX_ATEN_FALLBACK:
+            sym_registry.register_quantized_ops('caffe2', opset_version)
```

Comment:
you can change it to register_c2_ops, and register all ops in symoblics_caffe2, including overriding some regular ops' symbolics, such as relu, to support the quantized version.